1.8.9/src/main/java/io/github/axolotlclient/modules/zoom/Zoom.java

```diff
@@ -79,6 +81,12 @@ public static void update() {
 		} else if (shouldStop()) {
 			stop();
 		}
+
+		if(increase.wasPressed()){
+			scroll(zoomSpeed.get()/2);
+		} else if (decrease.wasPressed()){
+			scroll(-zoomSpeed.get()/2);
+		}
```

Comment:
```suggestion
		while (increase.wasPressed()) {
			scroll(zoomSpeed.get()/2);
		}
		while (decrease.wasPressed()) {
			scroll(-zoomSpeed.get()/2);
		}
```

Reply:
> 

are you sure this doesn't cause the game to freeze up?

Reply:
I'm pretty sure the game queues the events and wasPressed is true until a counter reaches 0.

Reply:
I was more concerned about the while loop honestly since it could get stuck in it for some amount of time and block further rendering

Reply:
it can't get stuck, it just returns true the number of times it was pressed